### PR TITLE
[RFR] Extract film plane from dicom

### PR DESCRIPTION
**Pull request by SimonBiggs**

### CODE REVIEW

**Comment by SimonBiggs** [comment]
Not all changes within the library here are tested unfortunately. At some point (not sure when) I can write more tests for this one if it is thought to need it...

File: packages/pymedphys_dicom/tests/test_structure.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            [5, 6, 3, 5],
+            [7, 7, 5, 7],
+            [3, 3, 3, 3]
+        ]
+    ]
+)
+
+
+def test_pull_structure():
+    dicom_structure = dicom_dataset_from_dict({
```
Comment (Centrus007) [inline packages/pymedphys_dicom/tests/test_structure.py]: I'm still a *little* nervous about relying solely on datasets created from `dicom_dataset_from_dict()` for these purposes, at least until we have built a super-solid level of confidence in that function.

Comment (SimonBiggs) [reply packages/pymedphys_dicom/tests/test_structure.py]: Fair point. I'd be very keen to build confidence as much as we can. I personally really like this test format...

**Comment by SimonBiggs** [comment]
Thanks Matt :) if you're happy with it make sure you do the merge so you're on the commit. :)

File: packages/pymedphys_dicom/tests/test_structure.py (hunk 1 above)
Comment (Centrus007) [reply packages/pymedphys_dicom/tests/test_structure.py]: Me too. Confidence is my only concern. Other than that, it is beautiful and flexible.

**Pull request merged**